### PR TITLE
ignore files starting with _ when loading cogs

### PR DESCRIPTION
**Pull request by vivax3794**
this allows the user of util files in the category folders if they are prefixed with _.﻿

### CODE REVIEW

**Review by ReneganRonin — APPROVED** [review]
Looks good to me.

File: bot/bot.py [hunk 1]
```diff
@@ -15,9 +15,9 @@ def load_extensions(self) -> None:
         """Load all the extensions in the exts/ folder."""
         logger.info('Start loading extensions from ./exts/')
         for extension in constants.EXTENSIONS.glob("*/*.py"):
+            if extension.name[0] == "_":
+                continue  # ignore files starting with _
```
Comment (Xithrius) [inline bot/bot.py:19]: Would it be more pythonic to do this?
```suggestion
            if extension.name.startswith("_"):
                continue  # ignore files starting with _
```

Comment (vivax3794) [reply bot/bot.py:19]: right of course,

**Review by Shivansh-007 — APPROVED** [review]
Looks good to me.

**Review by Xithrius — APPROVED** [review]
LGTM

**Pull request merged**